tools/install-build-deps

```diff
@@ -862,7 +862,7 @@ def Main():
       choices=['arm64', 'x64'],
       help='Override the autodetected build CPU architecture')
   parser.add_argument(
-      '--rust-toolchain',
+      '--rust',
```

Comment:
you'll need to change `args.rust_toolchain` to `args.rust` below too

Reply:
D'oh! Done
This is the classic "I can certainly make this 2 line change without trying it out" :)